src/models/StratumV1Client.ts

```
@@ -549,6 +551,17 @@ export class StratumV1Client {
             }
 
 
+            // Submit share to API
+            await this.shareSubmissionService.submitShare({
```

ESLint

[error] Delete `······`

(prettier/prettier)